core/scripts/keystone/src/88_gen_ocr3_config.go

```diff
@@ -1,219 +1,19 @@
 package src
 
 import (
-	"crypto/ed25519"
-	"encoding/hex"
-	"encoding/json"
-	"fmt"
-	"time"
-
-	"github.com/ethereum/go-ethereum/common"
-
-	"github.com/smartcontractkit/libocr/offchainreporting2plus/confighelper"
-	"github.com/smartcontractkit/libocr/offchainreporting2plus/ocr3confighelper"
-	"github.com/smartcontractkit/libocr/offchainreporting2plus/types"
-
 	helpers "github.com/smartcontractkit/chainlink/core/scripts/common"
-	"github.com/smartcontractkit/chainlink/v2/core/services/ocrcommon"
-	"github.com/smartcontractkit/chainlink/v2/core/services/relay/evm"
+	ksdeploy "github.com/smartcontractkit/chainlink/integration-tests/deployment/keystone"
 )
 
-type TopLevelConfigSource struct {
```

Comment:
@archseer this is refactored in the integration-tests. 

want to refactor the rest of these script to use common code from there